hw/ip/hmac/doc/checklist.md

```diff
@@ -199,18 +199,18 @@ Documentation | [DESIGN_DELTAS_CAPTURED_V3][]     | N/A         |
 Testbench     | [ALL_TODOS_RESOLVED][]            | Done        | [issue #385][] item 3 not related / Resolved: [pr #749][]
 Tests         | [X_PROP_ANALYSIS_COMPLETED][]     | Waived      |
 Regression    | [NIGHTLY_REGRESSION_AT_100][]     | Done        |
-Coverage      | [CODE_COVERAGE_AT_100][]          | In Progress | [#820][] pending. prim_fifo_sync revised [common_cov_excl.el][] HMAC excl to be added
+Coverage      | [CODE_COVERAGE_AT_100][]          | Done        | [pr #945][]. prim_fifo_sync revised [hmac_cov_excl.el][]
 Coverage      | [FUNCTIONAL_COVERAGE_AT_100][]    | Done        |
 Issues        | [NO_ISSUES_PENDING][]             | Done        |
 Code Quality  | [NO_TOOL_WARNINGS_THROWN][]       | Done        | Xcelium warnings on C code
 Integration   | [PRE_VERIFIED_SUB_MODULES_V3][]   | N/A         |
-Review        | Reviewer(s)                       | Not Started | @eunchan @sjgitty @sriyerg
-Review        | Signoff date                      | Not Started |
+Review        | Reviewer(s)                       | Done        | @eunchan @sjgitty @sriyerg
```

Comment:
If you want an official sign off on this, please convene a physical review of the
items; those names should be the ones as reviewers in that case. If you want
to finish off before Wednesday, you can use Tim as the other design reviewer.

Reply:
Thanks for the reminder and suggestion Scott. I will send out an invite for HMAC signoff review.